plugins/repositories/core/src/test/java/org/pentaho/di/ui/repo/controller/RepositoryConnectControllerTest.java

```diff
@@ -188,18 +188,18 @@ public void testConnectToRepository() throws Exception {
     verify( repository ).connect( null, null );
```

Comment:
Copyright date on this file needs to be updated.

Reply:
fixed